GTG/core/config.py

```diff
@@ -165,7 +165,7 @@ def get(self, option):
         if value is None and default_value is None:
             raise ValueError(
                 'No valid configuration value or default value was '
-                'found for %s in %s'.format(option, self._section_name))
+                f'found for {option} in {self._section_name}')
```

Comment:
Ruff originally called this a violation of [string-dot-format-extra-positional-arguments (F523)](https://docs.astral.sh/ruff/rules/string-dot-format-extra-positional-arguments/) and proposed to make it
```
'found for %s in %s'.format())
```
which was obviously not what the code is intended to be. The correct fix would have been
```
'found for {} in {}'.format(option, self._section_name))
```
but I figured it makes more sense to use an fstring.